From ddb4d49188c94ddac0c5a928cd48fcd57e2877de Mon Sep 17 00:00:00 2001
From: Antonio Borneo <antonio.borneo@foss.st.com>
Date: Mon, 17 Oct 2022 16:10:53 +0200
Subject: [PATCH 069/529] ARM: dts: stm32: add exti entry 43 for USBH_EHCI on
 stm32mp151

Add exti entry 43 used for USBH_EHCI wake up.
This event is mapped to GIC 75.

Signed-off-by: Antonio Borneo <antonio.borneo@foss.st.com>
Signed-off-by: Alexandre Torgue <alexandre.torgue@foss.st.com>
Change-Id: I1e81dabba6da65bc55725c08b79c35f4e90a39b3
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/274970
Reviewed-by: Antonio Maria BORNEO <antonio.borneo@st.com>
Reviewed-by: Amelie DELAUNAY <amelie.delaunay@foss.st.com>
Tested-by: Antonio Maria BORNEO <antonio.borneo@st.com>
Domain-Review: Amelie DELAUNAY <amelie.delaunay@foss.st.com>
---
 arch/arm/boot/dts/stm32mp151.dtsi | 1 +
 1 file changed, 1 insertion(+)

--- a/arch/arm/boot/dts/stm32mp151.dtsi
+++ b/arch/arm/boot/dts/stm32mp151.dtsi
@@ -1248,6 +1248,7 @@
 					<31 0 &intc GIC_SPI 53  IRQ_TYPE_LEVEL_HIGH>,
 					<32 0 &intc GIC_SPI 82  IRQ_TYPE_LEVEL_HIGH>,
 					<33 0 &intc GIC_SPI 83  IRQ_TYPE_LEVEL_HIGH>,
+					<43 0 &intc GIC_SPI 75  IRQ_TYPE_LEVEL_HIGH>,
 					<47 0 &intc GIC_SPI 93  IRQ_TYPE_LEVEL_HIGH>,
 					<48 0 &intc GIC_SPI 138 IRQ_TYPE_LEVEL_HIGH>,
 					<50 0 &intc GIC_SPI 139 IRQ_TYPE_LEVEL_HIGH>,
